frontend/src/components/ForgotPassword.js

```diff
@@ -0,0 +1,126 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import { Redirect, Link } from 'react-router-dom'
+import { Button, Grid, Paper, TextField, Typography } from '@material-ui/core'
+import { withStyles } from '@material-ui/core/styles'
+
+import ErrorMessage from './ErrorMessage'
+import { forgotPassword } from '../actions/auth'
+
+const style = (theme) => ({
+  root: {
+    background: 'linear-gradient(to right, #5390d9, #4ea8de)',
+    display: 'flex',
+    flexDirection: 'column',
+    minWidth: '100%',
+    minHeight: '100vh',
+    alignItems: 'center',
+    justifyContent: 'center'
+  },
+  paper: {
+    alignItems: 'center',
+    minWidth: '460px',
```

Comment:
I see a wider margin on the right side of the box and this is not consistent with the other UIs. The left and right margins should be the same and consistent with the others.
If you want the title to stay on one line, you could use `white-space: nowrap;` for that component.